woocommerce/woocommerce
# Add Business Location page to the core profiler

### Changes proposed in this Pull Request:



Closes https://github.com/woocommerce/team-ghidorah/issues/198 and [[Loading] - Create a loader component#199](https://github.com/woocommerce/team-ghidorah/issues/199)

This PR adds Business Location page. I've also worked on [[Loading] - Create a loader component#199](https://github.com/woocommerce/team-ghidorah/issues/199) in this PR since the loader needs a page for testing anyway.



### How to test the changes in this Pull Request:

1. Checkout this branch.
2. Go to the core profiler (OBW)
3. Click on `Skip guided tour`
4. Confirm `storeprofiler_step_view` track is recorded.
5. Select a country.
6. Click on `Go to my store` button.
7. Confirm `storeprofiler_step_completed` track is recorded.
8. Loader screen should show up for 3 seconds
9. Then you're redirected to WooCommerce Home

**Note on the loader page:**

The loader [configuration](https://github.com/woocommerce/woocommerce/pull/38019/files#diff-2944845107cf67e4f7c24f66369c78ea657b27b8816abcb406d26f4991304ee2R387) used in this PR is very simple as we only wait for 3 seconds and redirect to Woo Home. For multi-stage loader configuration, see https://github.com/woocommerce/woocommerce/commit/139889d9945603a21679dc3f4610848f88c64a1c. You can also checkout [demo/core-profiler-loader](https://github.com/woocommerce/woocommerce/compare/demo/core-profiler-loader) branch and perform the exact test instructions to see the demo.
